templates/base.html

```diff
@@ -126,8 +126,9 @@
             You can contribute to, or report problems with,
             <a class="p-link--external" href="https://bugs.launchpad.net/snapd">snapd</a>,
             <a class="p-link--external" href="https://bugs.launchpad.net/snapcraft">Snapcraft</a>,
+            <a class="p-link--external" href="https://forum.snapcraft.io/doc">documentation content</a>
```

Comment:
This link is 404. Did you mean `/c/doc`?

Reply:
d'oh! Fixed